adapters/handlers/rest/configure_api.go

```diff
@@ -148,6 +148,10 @@ func MakeAppState(ctx context.Context, options *swag.CommandLineOptionsGroup) *s
 		err := sentry.Init(sentry.ClientOptions{
 			Dsn:              appState.ServerConfig.Config.Sentry.DSN,
 			Debug:            appState.ServerConfig.Config.Sentry.Debug,
+			Release:          "weaviate@" + config.WeaviateVersion,
```

Comment:
you can just use `appState.ServerVersion` there's no need for `config.WeaviateVersion`

Reply:
Isn't this server version coming from the open api schema spec ? That means this version we can't use for preview build we might deploy right ?

Reply:
preview version consists of `PR title` and `git hash`, you construct `WeaviateVersion` from those 2 values, do you need a pr title in it? and a second thing is that in case of release you leave this setting as `unknown` that's OK, yes?